runtime/src/main/java/io/quarkus/reactivemessaging/http/runtime/QuarkusWebSocketConnector.java

```diff
@@ -80,6 +86,20 @@ public SubscriberBuilder<? extends Message<?>, Void> getSubscriberBuilder(Config
         Integer maxRetries = config.getMaxRetries();
         URI url = URI.create(config.getUrl());
 
-        return new WebSocketSink(vertx, url, serializer, serializerFactory, maxRetries, delay, jitter).sink();
+        TlsConfiguration tlsConfiguration = null;
+
+        // Check if we have a named TLS configuration or a default configuration:
+        if (config.getTlsConfigurationName().isPresent()) {
```

Comment:
Isn't the same code here and in the HttpSocketConnector?

Reply:
correct, moved to utility class